samples/SwiftDemoApp/SwiftDemoApp/MasterViewController.swift

```diff
@@ -32,24 +32,27 @@ class MasterViewController: UITableViewController {
 
   override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) ->
       UITableViewCell {
-    let cellIdentifier = "Cell"
-    var cell = tableView.dequeueReusableCell(withIdentifier: cellIdentifier)
-    if (cell == nil) {
-      cell = UITableViewCell(style: .subtitle, reuseIdentifier: cellIdentifier)
-      cell!.accessoryType = .disclosureIndicator
-    }
-
-    let sample = samples.object(at: indexPath.item) as! NSDictionary
-    cell!.textLabel!.text = sample.value(forKey: "title") as? String
-    cell!.detailTextLabel!.text = sample.value(forKey: "description") as? String
-
+        let cellIdentifier = "Cell"
```

Comment:
Can you adjust spacing to 2 spaces here to match the rest of the formatting for our Swift files?

Reply:
Yeah, I will.